src/test/java/dev/snowdrop/release/services/CPaaSConfigUpdateServiceTest.java

```diff
@@ -0,0 +1,190 @@
+    }
+
+    @Test
+    public void testReleaseDr() {
+        final String token = ConfigProvider.getConfig().getValue("gitlab.token", String.class);
+        final String user = ConfigProvider.getConfig().getValue("gitlab.user", String.class);
+        GitService.GitConfig config = service.buildGitConfig(RELEASE, user, token, Optional.of(PREVIOUS_RELEASE), Optional.of(ConfigProvider.getConfig().getOptionalValue("gitlab.repository", String.class).orElse(user + "/springboot")));
+        try {
+            git.initRepository(config);
+            Stream<File> files = service.updateCPaaSFiles(RELEASE, git.getConfig(config).getRepository().getWorkTree(), PREVIOUS_RELEASE, false, true);
+            Map<String, File> fileMap = files.collect(Collectors.toMap(file -> file.getName(), file -> file));
+            CPaaSReleaseFile releaseFile = factory.createCPaaSReleaseFromStream(new FileInputStream(fileMap.get(RELEASE_FILE_NAME)));
+            CPaaSRelease release = releaseFile.getRelease();
+            List<CPaaSPipelines> pipelines = release.getPipelines();
+            pipelines.get(1).getStages().forEach(stage -> {
+                if ("create-errata-tool-advisories".equalsIgnoreCase(stage.getName())) {
+                    assertFalse(stage.getEnabled());
+                }
+            });
+        } catch (IOException | ExecutionException | InterruptedException e) {
+            e.printStackTrace();
+            fail(e);
+        }
+    }
+
+    @Test
+    public void testReleaseErCr() {
+        final String token = ConfigProvider.getConfig().getValue("gitlab.token", String.class);
+        final String user = ConfigProvider.getConfig().getValue("gitlab.user", String.class);
+        GitService.GitConfig config = service.buildGitConfig(RELEASE, user, token, Optional.of(PREVIOUS_RELEASE), Optional.of(ConfigProvider.getConfig().getOptionalValue("gitlab.repository", String.class).orElse(user + "/springboot")));
+        try {
+            git.initRepository(config);
+            Stream<File> files = service.updateCPaaSFiles(RELEASE, git.getConfig(config).getRepository().getWorkTree(), PREVIOUS_RELEASE, true, true);
+            Map<String, File> fileMap = files.collect(Collectors.toMap(file -> file.getName(), file -> file));
+            CPaaSReleaseFile releaseFile = factory.createCPaaSReleaseFromStream(new FileInputStream(fileMap.get(RELEASE_FILE_NAME)));
+            CPaaSRelease release = releaseFile.getRelease();
+            List<CPaaSPipelines> pipelines = release.getPipelines();
+            pipelines.get(1).getStages().forEach(stage -> {
+                if ("create-errata-tool-advisories".equalsIgnoreCase(stage.getName())) {
+                    assertFalse(stage.getEnabled());
+                }
+            });
+        } catch (IOException | ExecutionException | InterruptedException e) {
+            e.printStackTrace();
+            fail(e);
+        }
+    }
+
+    @Test
+    public void createNewBranch() throws Throwable {
```

Comment:
Same remarks as previously:
- name more descriptive
- add exeption to the method signature

Reply:
Done.